src/main/java/org/opensearch/security/securityconf/impl/v6/ConfigV6.java

```diff
@@ -67,6 +68,9 @@ public static class Dynamic {
         public String hosts_resolver_mode = "ip-only";
         public String transport_userrname_attribute;
         public boolean do_not_fail_on_forbidden_empty;
+        public OnBehalfOf on_behalf_of = new OnBehalfOf() {
```

Comment:
Do we need this instantiation, I do not see a usage of `on_behalf_of`. Maybe I missed it

Reply:
Yes I removed it.